plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -376,6 +383,35 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 					}
 				}
 
+	if (devices & Debuginator_Mouse) {
+		CApiInputControllerPtr mouse = script_api->Input->mouse();
+		input_wrapper.controller = mouse;
+
+		if (api->any_released(input_wrapper.controller) != UINT_MAX) {
+			input_wrapper.time_since_pressed = -1;
+		}
+
+
+		if (debuginator_is_open(debuginator)) {
+			CApiVector3 scroll = input_wrapper.axis("wheel");
+			if (scroll.y != 0) {
```

Comment:
We catch all input, which is why this felt natural to us (so you could just scroll without paying attention to where the mouse is if you're not using it to play). However I agree that in general you should not take away control when it is open, so it _should_ check for the rect.